File: wxrk_android/app/src/main/java/com/wxrk/model/dashbord/DayWiseSummary.kt
package com.wxrk.model.dashbord

import com.google.gson.annotations.SerializedName


data class DayWiseSummary(

    @SerializedName("id") var id: Int? = null,
    @SerializedName("user_id") var userId: String? = null,
    @SerializedName("user_type") var userType: String? = null,
    @SerializedName("android_usage_log_id") var androidUsageLogId: String? = null,
    @SerializedName("app_summary_log_id") var appSummaryLogId: String? = null,
    @SerializedName("log_date") var logDate: String? = null,
    @SerializedName("wxrk_per_minute") var wxrkPerMinute: String? = null,
    @SerializedName("total_app_usage_time") var totalAppUsageTime: String? = null,
    @SerializedName("day_total_time") var dayTotalTime: String? = null,
    @SerializedName("day_idle_time") var dayIdleTime: String? = null,
    @SerializedName("wxrk_earned") var wxrkEarned: String? = null,
    @SerializedName("wxrk_spent") var wxrkSpent: String? = null,
    @SerializedName("wxrk_balance") var wxrkBalance: String? = null,
    @SerializedName("status") var status: String? = null,
    @SerializedName("created_at") var createdAt: String? = null,
    @SerializedName("updated_at") var updatedAt: String? = null,
    @SerializedName("watch_time") var watchTime : String? = null

)

File: wxrk_web/app/Models/DayWiseSummary.php
<?php
namespace App\Models;

use App\User;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class DayWiseSummary extends Model
{
    use SoftDeletes;

    protected $table = 'day_wise_summary';
    
    protected $fillable = [
        'user_id', 
        'user_type', 
        'android_usage_log_id', 
        'app_summary_log_id', 
        'log_date', 
        'wxrk_per_minute', 
        'total_app_usage_time', 
        'day_total_time', 
        'day_idle_time', 
        'watch_time', 
        'time_saved', 
        'time_saved_percentage', 
        'wxrk_earned', 
        'wxrk_spent', 
        'wxrk_balance', 
        'status'
    ];
    
    /**
     * The attributes that should be hidden for arrays.
     *
     * @var array
    */
    protected $hidden = [
        'deleted_at',
    ];

    public function user() {
        return $this->belongsTo(User::class);
    }

    public function androidUsageLog() {
        return $this->belongsTo(androidUsageLog::class);
    }

    public function appSummaryLog() {
        return $this->belongsTo(AppSummaryLog::class);
    }

}
